test/api/assignments_test.rb

```diff
@@ -46,7 +46,7 @@ def test_api_accepts_submission_attempt
   def test_api_accepts_submission_attempt_with_multi_file_solution
     Notify.stub(:everyone, nil) do
       Xapi.stub(:exists?, true) do
-        post '/user/assignments', {  key: alice.key, code: 'THE CODE', path: 'ruby/one/code.rb', 
+        post '/user/assignments', {  key: alice.key, code: 'THE CODE', path: 'ruby/one/code.rb',
```

Comment:
Line is too long. [96/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.